### DISCUSSION TITLE
How can the Microcks support a https url?

### DISCUSSION THREAD

**Discussion by touensure** [microcks/microcks#974] [Q&A] [answered]
Hi, I noticed that the url generated by mickrocks is always HTTP, but can Microcks support a HTTPS url?  Is there anybody who can help me with this question? Thanks a lot!

**Comment by lbroudoux** [comment, marked as answer]
Hi @touensure!

Microcks doesn't expose Https but relies on Kubernetes ingress controller or external reverse proxy to handle the TLS termination. So when deployed into a Kube cluster, the Helm chart and Operator install TLS `Ingresses`. Many options are available to either generate a self-signed certificate, reuse an already provisioned one or trigger the generation of a specific one depending on your deployment target.

**Reply by touensure** [reply]
thanks a lot for your kind help!
